src/bundle/Resources/config/services.yml

```diff
@@ -151,3 +152,23 @@ services:
             - {name: kernel.event_subscriber, priority: -250}
 
     EzSystems\EzPlatformAdminUiBundle\Templating\Twig\UserPreferencesGlobalExtension: ~
+
+    EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Provider\ContentTypeChoiceListProvider: ~
+
+    EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Provider\LanguageChoiceListProvider:
```

Comment:
Idea behind `services/` directory was that we put service definitions there to avoid `services.yml` getting bloated.